src/components/Breadcrumb/Breadcrumb.module.scss

```diff
@@ -0,0 +1,40 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:slnt,wght@-10..0,100..900");
+
+* {
+  font-family: Inter;
+  margin: 0;
+}
+.breadcrumb-wrapper {
+  padding: 1rem 1.5rem;
+  // background: var(--white);
+  background: white;
+  height: min-content;
+  text-transform: capitalize;
+
+  .breadcrumb {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+
+    .breadcrumb-item {
+      &:not(:last-child)::after {
+        content: "/";
+        padding: 0 0.5rem;
+        // color: var(--gray);
+        color: gray;
```

Comment:
use hex code for colors and there should be a separate file for color variables using scss